README.md

```diff
@@ -47,9 +46,11 @@ resource "aws_instance" "web" {
 ```
 module "ansible_provisioner" {
    source    = "github.com/cloudposse/tf_ansible"
-   arguments = ["--user=ubuntu"]
+
+   arguments = ["--user=ubuntu", "--inventory-file=../ansible/inventory"]
```

Comment:
We do not need `--inventory file` except we use `group_vars`.
It is an Ansible bug. https://github.com/ansible/ansible/issues/17969